src/mods/workers.js

```diff
@@ -0,0 +1,349 @@
+idrinth.workers = {
+  list: '###WORKERS###',
+  result: {
+       /**
+         * @param {string} list
+         * @returns {undefined}
+         */
+     tiers: function(list) {
```

Comment:
You could you short version for functions as well (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).